src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -81,28 +78,19 @@ public int getSize() {
         return size;
     }
 
-    private int hash(Node<K,V> node) {
-        if (node.key == null) {
-            return 0;
-        }
-        return node.hashCode() % table.length;
-    }
-
-    private int putIndex(Node<K,V> node) {
-        return Math.abs(hash(node));
+    private int getIndex(Node<K,V> node) {
+        return node.key == null ? 0 : Math.abs(node.hashCode() % table.length);
```

Comment:
u can delete your overrided hashCode() and use method hashCode() from Object
```suggestion
        return node.key == null ? 0 : Math.abs(node.key.hashCode() % table.length);
```